release/package/mgbuild.sh

```diff
@@ -93,6 +93,8 @@ print_help () {
 
   echo -e "\nbuild options:"
   echo -e "  --git-ref string              Specify git ref from which the environment deps will be installed (default \"master\")"
+  echo -e "  --rust-version number         Specify rustc and cargo version which be installed (default \"1.80\")"
+  echo -e "  --node-version number         Specify nodejs version which be installed (default \"20\")"
```

Comment:
Please move the default versions here into mgbuild.sh and remove them from all the individual per-OS Dockerfiles.